docs/pages/usage.md

```diff
@@ -0,0 +1,124 @@
+
+- [RawKNNRegressor](api/estimators/raw.md)
+- [EuclideanKNNRegressor](api/estimators/euclidean.md)
+- [MahalanobisKNNRegressor](api/estimators/mahalanobis.md)
+- [GNNRegressor](api/estimators/gnn.md)
+- [MSNRegressor](api/estimators/msn.md)
+
+These estimators can be used [like any other `sklearn` regressor](https://scikit-learn.org/stable/getting_started.html#fitting-and-predicting-estimator-basics):
+
+```python
+from sknnr import EuclideanKNNRegressor
+from sknnr.datasets import load_swo_ecoplot
+from sklearn.model_selection import train_test_split
+
+X, y = load_swo_ecoplot(return_X_y=True)
+X_train, X_test, y_train, y_test = train_test_split(X, y)
+est = EuclideanKNNRegressor(n_neighbors=3).fit(X_train, y_train)
+
+print(est.score(X_test, y_test)) 
+# 0.11496218649569434
+```
+
+In addition to their core functionality of fitting, predicting, and scoring, `sknnr` estimators offer a number of other features, detailed below.
+
+### Independent Scores and Predictions
+
+When an independent test set is not available, the accuracy of a kNN regressor can be estimated by comparing each sample in the training set to its second-nearest neighbor, i.e. the closest point *excluding itself*. All `sknnr` estimators set `independent_prediction_` and `independent_score_` attributes when they are fit, which store the predictions and scores of this independent evaluation.
+
+```python
+print(est.independent_score_)
+# 0.10243925752772305
+```
+
+### Retrieving Dataframe Indexes
+
+In `sklearn`, the `KNeighborsRegressor.kneighbors` method can identify the array index of the nearest neighbor to a given sample. Estimators in `sknnr` offer an additional parameter `return_dataframe_index` that allows neighbor plots to be identified directly by their index.
+
+```python
+X, y = load_swo_ecoplot(return_X_y=True, as_frame=True)
+est = est.fit(X, y)
+
+# Find the distance and dataframe index of the nearest neighbors to the first plot
+distances, neighbor_ids = est.kneighbors(X.iloc[:1], return_dataframe_index=True)
+
+# Preview the nearest neighbors by their dataframe index
+print(y.loc[neighbor_ids[0]])
+```
+
+|       |   ABAM_COV |   ABGRC_COV |   ABPRSH_COV |   ACMA3_COV |   ALRH2_COV |
+|------:|-----------:|------------:|-------------:|------------:|------------:|
+| 52481 |          0 |           0 |      39.3469 |           0 |           0 |
+| 60089 |          0 |           0 |      22.1199 |           0 |           0 |
+| 56253 |          0 |           0 |      22.8948 |           0 |           0 |
+
+!!! warning
+    An estimator must be fit with a `DataFrame` in order to use `return_dataframe_index=True`.
+
+### Y-Fit Data
+
+The [GNNRegressor](api/estimators/gnn.md) and [MSNRegressor](api/estimators/msn.md) estimators can be fit with `X` and `y` data, but they also accept an optional `y_fit` parameter. If provided, `y_fit` is used to fit the ordination transformer while `y` is used to fit the kNN regressor.
```

Comment:
Maybe we need an explanation of *why* you would use this? Not sure.

Reply:
I tried to expand on this section.  I felt like I was being a bit too domain-specific in my examples, but if we note that `sknnr` is really motivated by forest estimation questions, this feels OK.  Let me know if you think we should generalize a bit more than I've done.